### PR TITLE
Fix: Add layout width for content block

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

**Comment by coreyar** [comment]
These viz reg tests should not be marked as new

**Review by coreyar — APPROVED** [review]

**Comment by therealemjy** [comment]
Unfortunately this doesn't solve the issue:
Screen Shot 2022-04-20 at 08 20 27

Maybe park this for now. I think the issue has to do with the content of the dashboard rather than its container.

File: src/assets/styles/Global.scss [hunk 1]
```diff
@@ -31,6 +31,9 @@
   --color-border-main: #eaeaea;
   --color-primary: var(--color-yellow);
   --color-secondary: #d3d3d3;
+
+  /* Scrollbar width */
+  --scrollbar-width: 2px;
```
Comment (therealemjy) [inline src/assets/styles/Global.scss:36]: Let's not rely on our css files. In the future we'll ideally get rid of them completely.

**Review by therealemjy — CHANGES_REQUESTED** [review]

File: src/assets/styles/Global.scss (hunk 1 above)
Comment (baomastr) [reply src/assets/styles/Global.scss:36]: it was made just for understanding where we get this value (2px) for our new layout.

**Comment by baomastr** [comment]
> Maybe park this for now. I think the issue has to do with the content of the dashboard rather than its container.

I think, we should both avoid scrolling of layout and fix the content to get parent width. 
I'll add the overflow: hidden to avoid horizontal scrolling of layout. And then we should fix the tables component

**Comment by therealemjy** [comment]
I'd avoid both setting a fixed width and applying `overflow: hidden`.
The content should be stylized so it doesn't overflow, without the need for modifications on the parent container. If it was to overflow because of an issue, we'd want the scrollbar to be there so users can still see all the content and keep using the website (even if it wouldn't be the best UX, at least we wouldn't block them from being able to do actions).

**Comment by baomastr** [comment]
Okay then. Will close the PR and fix the tables component in new ticket.

**Pull request closed without merging**